### PR TITLE
Make custom application.yml additional.

### PR DESCRIPTION
**Pull request by clemens-tolboom**
See #488

### CODE REVIEW

File: armadillo/src/main/resources/application.yml [hunk 1]
```diff
@@ -31,7 +31,6 @@ spring:
       max-request-size: 1000MB
   application:
     name: armadillo
-  security:
```
Comment (mswertz) [inline armadillo/src/main/resources/application.yml:34]: this doesn't make sense to me?

**Review by DickPostma — APPROVED** [review]
LGTM

File: armadillo/src/main/resources/application.yml (hunk 1 above)
Comment (clemens-tolboom) [reply armadillo/src/main/resources/application.yml:34]: The whole file seems too much somehow. Logging is mixed with effort for PR #479

Comment (clemens-tolboom) [reply armadillo/src/main/resources/application.yml]: Restored `security:`

**Pull request merged**